access/handler.go

```diff
@@ -708,9 +708,9 @@ func (h *Handler) GetExecutionResultForBlockID(ctx context.Context, req *access.
 func (h *Handler) GetExecutionResultByID(ctx context.Context, req *access.GetExecutionResultByIDRequest) (*access.ExecutionResultByIDResponse, error) {
 	metadata := h.buildMetadataResponse()
 
-	blockID := convert.MessageToIdentifier(req.GetId())
+	resultId := convert.MessageToIdentifier(req.GetId())
```

Comment:
nit: the golang convention is to capitalize acronyms
```suggestion
	resultID := convert.MessageToIdentifier(req.GetId())
```